app/src/main/java/com/example/navadon/androidonclickevent/Option1Activity.java

```diff
@@ -44,4 +45,20 @@ private void hideKeyboardInput(View v){
         InputMethodManager imm = (InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE);
         imm.hideSoftInputFromWindow(v.getWindowToken(), 0);
     }
+
+    public void Back(View view) {
+        if(view.getId() == R.id.back) {
+            Intent intent = new Intent(this, Option0Activity.class);
+            startActivity(intent);
+            finish();
+        }
+    }
+
+    public void Next(View view) {
+        if(view.getId() == R.id.next) {
```

Comment:
You can actually merge methods "Back" and "Next" where you can merge the "if" blocks.